cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -27,6 +27,9 @@
 // DRM Card nodes start at 0
 #define DRM_CARD_NODE_START 0
 
+#define IVSH_WIDTH 1920
+#define IVSH_HEIGHT 1080
```

Comment:
The width/height is too common, just 1080p display mode. When connect with 1080p monitor, it should lead to some problem.